### DISCUSSION TITLE
Unable to create YouTube Client on blazor wasm

### DISCUSSION THREAD

**Discussion by dhairya-cse** [Tyrrrz/YoutubeExplode#652] [Q&A] [answered]
Hello, I am unable to create `YoutubeClient` on blazor wasm. Getting the following exception. Any ideas how to solve this?
```
System.PlatformNotSupportedException: Operation is not supported on this platform.
   at System.Net.Http.BrowserHttpHandler.set_UseCookies(Boolean value)
```

**Comment by Tyrrrz** [comment, marked as answer]
You could try to provide your own `HttpClient`

**Reply by dhairya-cse** [reply]
Ah Snap! Missed that even after looking at the constructor. Thanks a lot!
